packaging/systemd/microshift-aio.service

```diff
@@ -14,8 +14,7 @@ Environment=PODMAN_SYSTEMD_UNIT=%n
 Restart=on-failure
 TimeoutStopSec=70
 ExecStartPre=/bin/rm -f %t/%n.ctr-id
-ExecStartPre=/bin/bash /usr/bin/microshift-aio # not autogenerated
```

Comment:
While the volume check was unnecessary, the `microshift-aio` script also took care of ensuring that the `container_manage_cgroup` boolean was set. If you're dropping this script, should that functionality be rolled into the unit as another `ExecStart` directive, or is that simply going to become a documentation issue?

Reply:
we considered that, yes, and decided it would be a documentation issue. we can reconsider that, though - the thought was it's not the norm to manage such things (selinux booleans) in a unit file. definitely up for discussion, though!